packages/react-dom/src/__tests__/ReactServerRenderingHydration-test.js

```diff
@@ -598,4 +602,96 @@ describe('ReactDOMServerHydration', () => {
     expect(customElement.str).toBe(undefined);
     expect(customElement.obj).toBe(undefined);
   });
+
+  // @gate experimental || !www || !__DEV__
```

Comment:
Shouldn’t these experimental gates go too?

Reply:
These were part of the other PR, but no, these tests are actually testing the partial renderer. So these gates still apply until we get rid of the partial renderer. Basically it's only DEV www-classic.